containers/azure-functions-java-8/.devcontainer/Dockerfile

```diff
@@ -1,6 +1,15 @@
 # Find the Dockerfile for mcr.microsoft.com/azure-functions/java:3.0-java8-core-tools at this URL
 # https://github.com/Azure/azure-functions-docker/blob/master/host/3.0/buster/amd64/java/java8/java8-core-tools.Dockerfile
-FROM mcr.microsoft.com/azure-functions/java:3.0-java8-core-tools
+FROM mcr.microsoft.com/azure-functions/java:3.0-java11-core-tools
```

Comment:
Why java 8 depends on java11?

Reply:
VSCode java extension is only available on Java 11.  It is workaround, however, we want to keep project as 8 and want to use VSCode extension, we need to install both Java 11 and Java 8. 
I was consider if we change the auzre-functions-docker, however, including both images for the CI user is not good. So I add to install Java 8 later in the Dockerfile.